cli/tools/registry/mod.rs

```diff
@@ -426,21 +441,21 @@ impl PublishPreparer {
 
       let exports_content = format!(
         r#"{{
-  "name": "{}",
-  "version": "{}",
-  "exports": "{}"
-}}"#,
+      "name": "{}",
+      "version": "{}",
+      "exports": "{}"
+    }}"#,
```

Comment:
This is introducing a change in the message. Surprisingly we don't have a test for this to have caught this.